peewee_extra_fields/__init__.py

```diff
@@ -1153,6 +1153,32 @@ def python_value(self, value: str) -> namedtuple:
         return value
 
 
+class EnumField(CharField):
+    """
+    This class enables a Enum like field for Peewee.
+    """
+    db_field = "enum"
+
+    def __init__(self, choices, *args, **kwargs):
+        if not issubclass(choices, Enum):
```

Comment:
@juancarlospaco Do you agree with this explicit type checking here?

Reply:
I think it also needs to check for `enum.IntEnum` see https://devdocs.io/python~3.6/library/enum#enum.IntEnum

So, it should be something like:
```python
if not issubclass(choices, (Enum, IntEnum)):
```